microcosm_flask/encryption/conventions/crud_adapter.py

```diff
@@ -0,0 +1,41 @@
+from microcosm_flask.conventions.crud_adapter import CRUDStoreAdapter
+
+
+class EncryptableCRUDStoreAdapter(CRUDStoreAdapter):
+    """
+    Adapt the CRUD conventions callbacks to the `EncryptableStore` interface.
+
+    """
+
+    def update(self, **kwargs):
```

Comment:
Maybe it's preferable to call this: update_and_reencrypt.
In some cases, we want to use the update function again and again without reencrypting the data.

Reply:
Okay.